FinniversKit/Sources/Recycling/GridViews/AdRecommendations/Cell/ExternalAdRecommendationCell.swift

```diff
@@ -151,13 +152,19 @@ public class ExternalAdRecommendationCell: UICollectionViewCell, AdRecommendatio
 
         backgroundColor = .clear
 
+        let imageHeightMinimumConstraint = imageContentView.heightAnchor.constraint(equalTo: imageContentView.widthAnchor, multiplier: ExternalAdRecommendationCell.minImageAspectRatio)
+        let imageHeightMaximumConstraint = imageContentView.heightAnchor.constraint(lessThanOrEqualTo: imageContentView.widthAnchor, multiplier: ExternalAdRecommendationCell.maxImageAspectRatio)
+
+        imageHeightMinimumConstraint.priority = .defaultHigh
```

Comment:
I wonder why we set these min and max height constraints, instead of setting the actual height based on image size when configuring the cell with the model 🤔 But I see we do the same in `StandardAdRecommendationCell`, so if it works there it should work here as well 👍

Reply:
when we got a new report, i decided to revert my previous changes [from here](https://github.com/finn-no/FinniversKit/pull/1324) and attempt to fix it from scratch
then i noticed that the label had a constraint missing! added it and it seemed to make auto layout happy!
now i wonder why i did not see it last time...

Reply:
awesome 👍 haha not easy to see this stuff 👁️